t/lib/t/MusicBrainz/Server/Edit/Relationship/Create.pm

```diff
@@ -185,6 +182,7 @@ sub _create_edit {
         end_date => { year => 1995 },
         attributes => [ ],
         ended => 1,
+        privileges => $UNTRUSTED_FLAG,
```

Comment:
Yes, for reference see https://github.com/metabrainz/musicbrainz-server/blob/65f54a1247f0c1003b7dc9cc0a47b96463b97794/lib/MusicBrainz/Server/Data/Edit.pm#L484

This is also how the "make votable" checkbox works: https://github.com/metabrainz/musicbrainz-server/blob/65f54a1247f0c1003b7dc9cc0a47b96463b97794/lib/MusicBrainz/Server/Controller.pm#L83